### PR TITLE
1166 Improve download report step

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
Makes things look nice and gives user more information about report that will be downloaded

### CODE REVIEW

File: afs/templates/views/components/workflows/project-report-workflow/report-template-select.htm [hunk 1]
```diff
@@ -62,11 +62,9 @@
                 data-bind="css: { 
                         'active': $parent.templateValue() === id 
                     },
-                    click: function () {
```
Comment (aarongundel) [inline afs/templates/views/components/workflows/project-report-workflow/report-template-select.htm:65]: Why did this get changed?  This was more correct the first time (wrapping it in anonymous function - which ensures the function is not called on initialization; you assign a function vs calling that function) - if you don't do this, you'll get a default value set for the template - which may have been your intention, but if you do it like this, template name doesn't get set - and you get [Object object] in the "summary/download" step.  Try this with a single template in your db and you'll see what I mean.

**Review by aarongundel — CHANGES_REQUESTED** [review]
Mostly ok; one issue with a single template workflow.  Also can you purge the "sectionTitle"s from project-report-workflow.js?

**Review by aarongundel — CHANGES_REQUESTED** [review]
Mostly ok; one issue with a single template workflow.  Also can you purge the "sectionTitle"s from project-report-workflow.js?

File: afs/templates/views/components/workflows/project-report-workflow/report-template-select.htm (hunk 1 above)
Comment (johnatawnclementawn) [reply afs/templates/views/components/workflows/project-report-workflow/report-template-select.htm:65]: This was an artifact of me trying to debug passing values between workflow steps. I think everything should be sorted now.

**Review by aarongundel — APPROVED** [review]
LGTM

**Pull request merged**